build/tasks/ros.sh

```diff
@@ -104,3 +104,11 @@ task:rosversion() {
 task:rqt_graph() {
   task:roscommand "rqt_graph ${@}"
 }
+
```

Comment:
Putting this in another PR might be a little cleaner, but keeping bureaucratic overhead low is also important of course

Reply:
Since I was using rqt_plot often, I just added this command.